meshtastic/node_runtime/channel_export_runtime.py

```
@@ -0,0 +1,231 @@
+            )
+
+        local_config_snapshot = self._snapshot_local_config()
+        if not local_config_snapshot.HasField("lora"):
+            self._node.requestConfig(
+                local_config_snapshot.DESCRIPTOR.fields_by_name["lora"]
+            )
+            wait_for_config = getattr(self._node, "waitForConfig", None)
+            if callable(wait_for_config):
+                result = wait_for_config(attribute="lora")
+                if not result:
+                    raise RuntimeError("Timeout waiting for LoRa config")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)